plugin/pkg/auth/authenticator/token/bootstrap/bootstrap.go

```diff
@@ -150,7 +150,7 @@ func (t *TokenAuthenticator) AuthenticateToken(token string) (user.Info, bool, e
 	}, true, nil
 }
 
-// Copied from k8s.io/client-go/tools/bootstrap/token/api
+// Copied from k8s.io/bootstrap/token/api
```

Comment:
why is this copied?

Reply:
i'm not exactly sure why it was copied before. this PR moves the files from client-go to bootstrap, without touching the files' functions. i would leave it to the original author to fix the copy if necessary.

Reply:
@luxas @timstclair you probably know